.github/workflows/release.yml

```diff
@@ -100,6 +100,80 @@ jobs:
           name: crystaldba_cli-artifacts
           path: ./agent/dist/*
 
+  build_linux_amd64:
+    needs: prepare
+    runs-on: ubuntu-20.04
+    defaults:
+      run:
+        working-directory: ./agent
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+
+      - name: Install Poetry
+        uses: abatilo/actions-poetry@v2
+        with:
+          poetry-version: "2.0.1"
+
+      - name: Build
+        run: poetry install
+
+      - name: Build with Nuitka (Linux AMD64)
+        run: poetry run package_amd64
+
+      - name: Prepare Tar Archive
+        run: |
+          tar -czf ./build/linux_amd64/crystal-client-linux-amd64-${{ needs.prepare.outputs.release_tag }}.tar.gz -C ./build/linux_amd64 main.dist
```

Comment:
The install.sh script generates a filename with underscores not hyphens. I think we want hyphens but in any case we need consistency.